runtime/interface.go

```diff
@@ -202,10 +202,6 @@ func (i *emptyRuntimeInterface) RemoveAccountKey(_ Address, _ int) (publicKey []
 	return nil, nil
 }
 
-func (i *emptyRuntimeInterface) UpdateAccountCode(_ Address, _ []byte) error {
```

Comment:
Not sure how this change connects to the rest of the changes.

Reply:
Oh, this is unrelated. I saw it was old, unused code, then accidentally committed the change as part of this PR.

Reply:
Reverted in 3ccad68